cif_pow.dic

```diff
@@ -11885,6 +11957,24 @@ save_pd_spec.orientation
 
 save_
 
+save_pd_spec.prep_id
+
+    _definition.id                '_pd_spec.prep_id'
+    _definition.update            2023-06-04
+    _description.text
+;
+    A prep id, representing a sample, to which this specimen relates.
```

Comment:
"The identifier for the sample from which this specimen was taken." is better.

Reply:
Greatly